cpp/notebooks/cpp_pcl-1.10.ipynb

```diff
@@ -1,23 +1,21 @@
 {
  "cells": [
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "80bc1cb4",
+   "cell_type": "markdown",
+   "id": "30ace377",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "! if [ ! -d boost_git ]; then git clone -b boost-1.71.0 --single-branch --recurse-submodules https://github.com/boostorg/boost.git boost_git; fi"
+    "### For Ubuntu 20.04"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c8fba46e",
+   "id": "90029f82",
    "metadata": {},
    "outputs": [],
    "source": [
-    "! ../scripts/generate_cling_boost.py boost_git"
+    "! ../scripts/generate_cling_boost.py "
```

Comment:
let's change this for all the notebook too

Reply:
Done.